### DISCUSSION TITLE
Possibility to have refiner such as SliderGraphBar

### DISCUSSION THREAD

**Discussion by MaximeChofardet** [microsoft-search/pnp-modern-search#3005] [Ideas]
Would like to know if a workaround is available or has been implemented to reproduced previous refiner Template  SliderGraphBar.
This template previously allowed to have such refiner  : 
![Capture](https://user-images.githubusercontent.com/36511028/93915156-e19d4400-fd07-11ea-86d7-45d34e77394d.PNG)

**Comment by wobba** [comment]
We accept PR's and you also need to be able to specify the buckets for this to work properly. What's your usecase?

**Comment by MaximeChofardet** [comment]
Our project is to make a redesign of an existing search engine already realized on SharePoint Online but on publishing site and all the related search webpart. The previous search engine was composed of 7 distinct pages configured according the managed languages and used to have all display templates that incorporated custom design.
In those pages, users where able to search by keyword and filtrer the results by different metadata but almost one which is a number and equal to contract turnover.
This refiner used the display template SliderGraphBar.js and allowed user tu have a graph bar représentation of the results (occurrence nb) according to range that could be set manually in the settings of the webpart. After click on one of the bar, results was filter by metadata "turnover" between X and Y value.
Our objective is to have a similar refiner template available on the modern search

**Comment by wobba** [comment]
@MaximeChofardet do you have numbers showing people were using this slider template refiner, or if a different date refiner would solve the job equally well? I'm not saying it's not useful, just that someone would have time to program it justifying the need.

**Comment by MaximeChofardet** [comment]
@wobba , the site is used by 27 000 users

**Comment by wobba** [comment]
How many people are using a site does not say anything about feature usage :) So, I guess you have no custom telemetry or user research pointing to the real value of such a slider :)

**Comment by MaximeChofardet** [comment]
Hum you are right concerning the difference between the site usage / feature usage and indeed no telemetry is applied on it.
To be honest, would like to know who succeeded in getting a client validation (with cost allocation)  to implement a usage telemetry on a particular type of search filter.
What I know is that this feature is the main one on the search pages because it allows to business consultants , before a commercial approach with a third party customer, to identifiy according to the deals size what can be the closest reference already realized intuitively.

**Comment by wobba** [comment]
You'd be surprised on what telemetry people hook into the search page. That said, for v4 we will probably support templating for refiners/filter as well, which means someone could develop this and share with the community.
